packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -12,6 +12,7 @@
 
 import {AriaLabelingProps, DOMProps, DOMRef, StyleProps} from '@react-types/shared';
 import {classNames, SlotProvider, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import * as DragManager from '@react-aria/dnd/src/DragManager';
```

Comment:
This import will not work in the built code

Reply:
how do we not have a lint rule for this...